debian/control

```diff
@@ -6,6 +6,7 @@ Rules-Requires-Root: no
 Build-Depends:
  debhelper-compat (= 13),
  cmake,
+ pkg-config,
```

Comment:
`lintian` gives off a warning that

```
W: igsc source: build-depends-on-obsolete-package Build-Depends: pkg-config => pkgconf
N: 
N:   The package build-depends on a package that has been superseded. If the superseded package is part of an ORed group, it should not be the first package in the group.
N: 
N:   Visibility: warning
N:   Show-Always: no
N:   Check: fields/package-relations
N: 
N:
```

So please update the dependency to `pkgconf` instead.